File: Javascript/7-Kyu/Regex-validate-PIN-code.js
// Regex validate PIN code
function validatePIN (pin) {

    let regex = /^\d{4}$|^\d{6}$/;
    return regex.test(pin);

};

// Tests
console.log(validatePIN("1234"))   // -->  true
console.log(validatePIN("12345"))  // -->  false
console.log(validatePIN("a234"))   // -->  false

File: Python/7-Kyu/Regex-validate-PIN-code.py
# Regex validate PIN code
def validate_pin(pin):
    import re

    return True if re.search('^\d{4}$|^\d{6}$', pin) else False

# Tests
print(validate_pin("1234"))   # -->  true
print(validate_pin("12345"))  # -->  false
print(validate_pin("a234"))   # -->  false
